src/db.rs

```diff
@@ -185,8 +185,13 @@ pub fn add_yaml_to_room(
     let game_name = match &parsed.game {
         YamlGame::Name(name) => name.clone(),
         YamlGame::Map(map) => {
-            if map.len() == 1 {
-                map.keys().next().unwrap().clone()
+            let weighted_map: HashMap<&String, &f64> =
+                map.iter().filter(|(_, &weight)| weight >= 1.0).collect();
+
+            if weighted_map.len() == 1 {
+                weighted_map.keys().next().unwrap().to_string()
+            } else if weighted_map.len() > 1 {
+                format!("Random ({})", weighted_map.len())
             } else {
```

Comment:
Pretty sure branch here is not attainable here unless the YAML contains 0 game. We should probably return an error instead (and remove the unwrap on the caller, no idea why I never changed that :rofl:)

Reply:
Left it just in case™️ but as mentioned the filter and error handling would best take place in `upload_yaml()`. That filtered list could be reused in the room template when hovering over the Random entry.

Reply:
Yeah I was thinking about that at the beginning but then if someone uploads a 20+ games YAML it's going to look like crap.